packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 49.6.0 - August 20, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Remove files in an existing project when running the `SFDX: Create and Set Up Project for ISV Debugging` command ([PR #2407](https://github.com/forcedotcom/salesforcedx-vscode/pull/2407))
+
+#### salesforcedx-sobjects-faux-generator
+
+- Fix SObject refresh to ignore entities ending with Share, History, Feed and Event ([PR #2412](https://github.com/forcedotcom/salesforcedx-vscode/pull/2412))
+
+## Added
```

Comment:
Everything under added can be removed. The SLDS PR was just an update to the readme and the deploy PR has no change in functionality we only updated to a newer version of the deploy-retrieve library